hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/Entity.java

```diff
@@ -72,27 +74,89 @@ public final String toString() {
      * the subclasses are free to use a blueprint type not inheriting from this one.
      */
     public abstract static class Blueprint extends AbstractElement.Blueprint {
-        @XmlAttribute
         private final String id;
+        private final Map<String, Set<CanonicalPath>> outgoing;
```

Comment:
But because I forgot that I actually added this in, I didn't add tests for it and *gasp* I didn't even implement it ;)